sourcecode/report_generator.py

```diff
@@ -0,0 +1,167 @@
+import calendar
+
+
+class ReportGenerator():
```

Comment:
Avoid using the word `generator` since the generator is a feature of Python so let's not mix it.
We can say it `WeatherReportsDisplay`